src/main/java/de/tum/in/www1/artemis/service/connectors/localvc/LocalVCPersonalAccessTokenManagementService.java

```diff
@@ -0,0 +1,43 @@
+package de.tum.in.www1.artemis.service.connectors.localvc;
+
+import static de.tum.in.www1.artemis.config.Constants.PROFILE_LOCALVC;
+
+import java.security.SecureRandom;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+
+import de.tum.in.www1.artemis.web.rest.UserResource;
+
+@Service
+@Profile(PROFILE_LOCALVC)
+public class LocalVCPersonalAccessTokenManagementService {
+
+    private static final Logger log = LoggerFactory.getLogger(UserResource.class);
+
+    @Value("${artemis.version-control.version-control-access-token:#{false}}")
+    private boolean versionControlAccessToken;
+
+    @Value("${artemis.version-control.vc-access-token-max-lifetime-in-days:365}")
+    private int vcMaxLifetimeInDays;
```

Comment:
This field isn't used anywhere

Reply:
:+1: I removed it, if we re-add this functionality later, we can also re-add the variables